src/runtime/storageNG/storage-proxy.ts

```diff
@@ -102,13 +110,14 @@ export class StorageProxy<T extends CRDTTypeRecord> {
       // If a handle configured for sync notifications registers after we've received the full
       // model, notify it immediately.
       if (handle.options.notifySync && this.synchronized) {
-        handle.onSync();
+        this.notifySync();
```

Comment:
this is going to notify all handles instead of just the new one, is it a problem?

Reply:
Yeah it might be.. fixed.